### PR TITLE
LLM: Modify CPU Installation Command for examples

### PR DESCRIPTION
**Pull request by Romanticoseu**
## Description

Base on #11042

### CODE REVIEW

File: python/llm/example/CPU/Applications/hf-agent/README.md [hunk 1]
```diff
@@ -9,11 +9,22 @@ To run this example with IPEX-LLM, we have some recommended requirements for you
 
 ### 1. Install
 We suggest using conda to manage environment:
+
+On Linux
```
Comment (xiangyuT) [inline python/llm/example/CPU/Applications/hf-agent/README.md:13]: On Linux:

File: python/llm/example/CPU/Applications/streaming-llm/README.md [hunk 2]
```diff
@@ -9,10 +9,20 @@ model = AutoModelForCausalLM.from_pretrained(model_name_or_path, load_in_4bit=Tr
...
+
+On Linux
 ```bash
 conda create -n llm python=3.11
 conda activate llm
 
+pip install --pre --upgrade ipex-llm[all] --extra-index-url https://download.pytorch.org/whl/cpu
+```
+
+On windows:
```
Comment (xiangyuT) [inline python/llm/example/CPU/Applications/streaming-llm/README.md:21]: Windows

File: python/llm/example/CPU/HF-Transformers-AutoModels/Advanced-Quantizations/GGUF/README.md [hunk 3]
```diff
@@ -24,19 +24,32 @@ In the example [generate.py](./generate.py), we show a basic use case to load a
...
 
 After installing conda, create a Python environment for IPEX-LLM:
+
+On Linux
 ```bash
 conda create -n llm python=3.11 # recommend to use Python 3.11
 conda activate llm
 
+pip install --pre --upgrade ipex-llm[all] --extra-index-url https://download.pytorch.org/whl/cpu
+ # install the latest ipex-llm nightly build with 'all' option
```
Comment (xiangyuT) [inline python/llm/example/CPU/HF-Transformers-AutoModels/Advanced-Quantizations/GGUF/README.md:34]:
```
pip install --pre --upgrade ipex-llm[all] --extra-index-url https://download.pytorch.org/whl/cpu # install the latest ipex-llm nightly build with 'all' option
```

If the length is too long, maybe move the comment to the front? 
```
# Install the latest ipex-llm nightly build with 'all' option.
pip install --pre --upgrade ipex-llm[all] --extra-index-url https://download.pytorch.org/whl/cpu
```

File: python/llm/example/CPU/HF-Transformers-AutoModels/Model/codegemma/README.md [hunk 4]
```diff
@@ -10,10 +10,26 @@ In the example [generate.py](./generate.py), we show a basic use case for a Code
...
 After installing conda, create a Python environment for IPEX-LLM:
+
+On Linux:
+
 ```bash
 conda create -n llm python=3.11 # recommend to use Python 3.11
 conda activate llm
 
+# install ipex-llm with 'all' option
+pip install ipex-llm[all] --extra-index-url https://download.pytorch.org/whl/cpu
```
Comment (xiangyuT) [inline python/llm/example/CPU/HF-Transformers-AutoModels/Model/codegemma/README.md:21]: pip install --pre --upgrade ipex-llm[all] --extra-index-url https://download.pytorch.org/whl/cpu

File: python/llm/example/CPU/HF-Transformers-AutoModels/Model/codegemma/README.md [hunk 5]
```diff
@@ -10,10 +10,26 @@ In the example [generate.py](./generate.py), we show a basic use case for a Code
...
+```
+
+On Windows:
+
+```cmd
+conda create -n llm python=3.11 # recommend to use Python 3.11
+conda activate llm
+
 # install ipex-llm with 'all' option
 pip install ipex-llm[all]
```
Comment (xiangyuT) [inline python/llm/example/CPU/HF-Transformers-AutoModels/Model/codegemma/README.md:34]: pip install --pre --upgrade ipex-llm[all]

File: python/llm/example/CPU/HF-Transformers-AutoModels/Model/moss/README.md [hunk 6]
```diff
@@ -9,10 +9,22 @@ To run these examples with IPEX-LLM, we have some recommended requirements for y
...
 ### 1. Install
 We suggest using conda to manage environment:
+
+On Linux:
+
 ```bash
 conda create -n llm python=3.11
 conda activate llm
 
+pip install ipex-llm[all] --extra-index-url https://download.pytorch.org/whl/cpu # install ipex-llm with 'all' option
```
Comment (xiangyuT) [inline python/llm/example/CPU/HF-Transformers-AutoModels/Model/moss/README.md:19]: --pre --upgrade

File: python/llm/example/CPU/HF-Transformers-AutoModels/Model/mpt/README.md [hunk 7]
```diff
@@ -8,10 +8,22 @@ To run these examples with IPEX-LLM, we have some recommended requirements for y
 In the example [generate.py](./generate.py), we show a basic use case for an MPT model to predict the next N tokens using `generate()` API, with IPEX-LLM INT4 optimizations.
 ### 1. Install
 We suggest using conda to manage environment:
+
+On Linux:
 ```bash
 conda create -n llm python=3.11
 conda activate llm
 
+pip install ipex-llm[all] --extra-index-url https://download.pytorch.org/whl/cpu # install ipex-llm with 'all' option
```
Comment (xiangyuT) [inline python/llm/example/CPU/HF-Transformers-AutoModels/Model/mpt/README.md:17]: same

File: python/llm/example/CPU/HF-Transformers-AutoModels/Model/cohere/README.md [hunk 8]
```diff
@@ -8,10 +8,23 @@ To run these examples with IPEX-LLM, we have some recommended requirements for y
...
 We suggest using conda to manage environment:
+
+On Linux:
+
 ```bash
 conda create -n llm python=3.11
 conda activate llm
 
+pip install --pre --upgrade ipex-llm[all] --extra-index-url https://download.pytorch.org/whl/cpu # install ipex-llm with 'all' option
+pip install tansformers==4.40.0
```
[the remark this answers is not in the document]
Comment (Romanticoseu) [reply python/llm/example/CPU/HF-Transformers-AutoModels/Model/cohere/README.md:19]: Also in six files, fixed.
image

File: python/llm/example/CPU/HF-Transformers-AutoModels/Advanced-Quantizations/GGUF/README.md [hunk 9]
```diff
@@ -24,19 +24,33 @@ In the example [generate.py](./generate.py), we show a basic use case to load a
...
 
 After installing conda, create a Python environment for IPEX-LLM:
+
+On Linux
 ```bash
 conda create -n llm python=3.11 # recommend to use Python 3.11
 conda activate llm
 
-pip install --pre --upgrade ipex-llm[all] # install the latest ipex-llm nightly build with 'all' option
+# install the latest ipex-llm nightly build with 'all' option
```
Comment (xiangyuT) [inline python/llm/example/CPU/HF-Transformers-AutoModels/Advanced-Quantizations/GGUF/README.md:33]: Should we align this comment position?

Comment (Romanticoseu) [reply python/llm/example/CPU/HF-Transformers-AutoModels/Advanced-Quantizations/GGUF/README.md:33]: Will fix all errors like this

File: python/llm/example/CPU/PyTorch-Models/Model/meta-llama/README.md [hunk 10]
```diff
@@ -20,7 +23,24 @@ git apply < ../cpu.patch # apply cpu version patch
...
+conda create -n llm python=3.11
+conda activate llm
+
+# Install meta-llama repository
+git clone https://github.com/facebookresearch/llama.git
+cd llama/
+git apply < ../cpu.patch # apply cpu version patch
+pip install -e .
+
+cd -
```
Comment (xiangyuT) [inline python/llm/example/CPU/PyTorch-Models/Model/meta-llama/README.md:42]: `cd -` can not work on Windows?

File: python/llm/example/CPU/PyTorch-Models/Model/mixtral/README.md [hunk 11]
```diff
@@ -12,18 +12,34 @@ In the example [generate.py](./generate.py), we show a basic use case for a Mixt
...
+
 ```bash
 conda create -n llm python=3.11 # recommend to use Python 3.11
 conda activate llm
 
 # below command will install PyTorch CPU as default
 pip install torch==2.0.1 --index-url https://download.pytorch.org/whl/cpu
-pip install --pre --upgrade ipex-llm[all]
+pip install --pre --upgrade ipex-llm[all] --extra-index-url https://download.pytorch.org/whl/cpu
+
```
Comment (Oscilloscope98) [inline python/llm/example/CPU/PyTorch-Models/Model/mixtral/README.md:25]: Looks strange for me. I am not sure whether `torch` with version `2.0.1` is required. If not, we could remove the `pip install torch` part as it has been included into our `ipex-llm[all]`

Comment (Oscilloscope98) [reply python/llm/example/CPU/PyTorch-Models/Model/mixtral/README.md:25]: @qiyuangong What do you think :)?

File: python/llm/example/GPU/HF-Transformers-AutoModels/Model/mpt/README.md [hunk 12]
```diff
@@ -20,14 +20,14 @@ pip install einops  # additional package required for mpt-7b-chat and mpt-30b-ch
 
 #### 1.2 Installation on Windows
 We suggest using conda to manage environment:
-```bash
+```cmd
```
Comment (Oscilloscope98) [inline python/llm/example/GPU/HF-Transformers-AutoModels/Model/mpt/README.md:23]: No need to change this readme in GPU example

File: python/llm/example/CPU/PyTorch-Models/Model/mixtral/README.md [hunk 13]
```diff
@@ -12,18 +12,34 @@ In the example [generate.py](./generate.py), we show a basic use case for a Mixt
...
 pip install transformers==4.36.0
 ```
 
+On windows:
+
+```cmd
+conda create -n llm python=3.11
+conda activate llm
+
+pip install torch==2.0.1 --index-url https://download.pytorch.org/whl/cpu
```
Comment (Oscilloscope98) [inline python/llm/example/CPU/PyTorch-Models/Model/mixtral/README.md:35]: Forgot to remove torch here

File: python/llm/example/GPU/HF-Transformers-AutoModels/Model/mpt/README.md (hunk 12 above)
Comment (Oscilloscope98) [reply python/llm/example/GPU/HF-Transformers-AutoModels/Model/mpt/README.md]: No need to change anything in this file :)

**Review by Oscilloscope98 — APPROVED** [review]
LGTM. May further update missing CPU examples in the next PR

File: python/llm/example/CPU/PyTorch-Models/Model/qwen1.5/README.md [hunk 14]
```diff
@@ -10,24 +10,41 @@ In the example [generate.py](./generate.py), we show a basic use case for a Qwen
...
+On Windows:
+
+```cmd
+conda create -n llm python=3.11
+conda activate llm
+
+pip install --pre --upgrade ipex-llm[all]
+pip install transformers==4.37.0
+
+pip install transformers==4.40.0
```
Comment (xiangyuT) [inline python/llm/example/CPU/PyTorch-Models/Model/qwen1.5/README.md:38]: Need to comment for Qwen1.5-MoE-A2.7B?

**Review by qiyuangong — APPROVED** [review]
LGTM

**Review by xiangyuT — APPROVED** [review]
LGTM

**Pull request merged**